pkg/stack_mgr/stack_service.go

```diff
@@ -173,20 +169,17 @@ func (s *StackService) Add(ctx context.Context, stackName string) (*Stack, error
 
 	// force refresh list of stacks, and add to it the new stack
 	s.stacks = nil
-	existStackNames := map[string]bool{}
 	existStacks, err := s.GetStacks(ctx)
 	if err != nil {
 		return nil, err
 	}
 
-	for name := range existStacks {
-		existStackNames[name] = true
-	}
-	existStackNames[stackName] = true
 	newStackNames := []string{}
-	for name := range existStackNames {
```

Comment:
`existStackNames` was never used aside from the generation of `newStackNames`, and this simplifies that by removing the intermediate map